apps/explorer/src/app/config/env.ts

```diff
@@ -0,0 +1,34 @@
+const windowOrDefault = (key: string) => {
+  if (window._env_ && window._env_[key]) {
+    return window._env_[key] as string;
+  }
+  return (process.env[key] as string) || '';
+};
+
+const truthy = ['1', 'true'];
+
+export const ENV = {
+  // Data sources
+  // Environment
+  nodeEnv: windowOrDefault('NODE_ENV'),
```

Comment:
i think some env variables are build time only by nature, like the NODE_ENV here, for example. this is usually supposed to describe the node environment the bundle was built in, which we can't really change from within the runtime.